components/Hero/src/index.scss

```diff
@@ -8,7 +8,8 @@
   position: relative;
 }
 
-.denhaag-hero--routing {
+.denhaag-hero--routing,
+.denhaag-hero--theme-page {
```

Comment:
Personally i'm more a fan of using `--theme` instead of `--theme-page` since a hero is kinda always used on a page ;-) Otherwise the other hero's must be modified with `-page` behind it.

@bddjong what are your thoughts?

Reply:
I think this is actually a mistranslation regardless. I had to run the storybook to understand what kind of variant we were actually talking about. For "thema paginas", thema refers to a topic or subject rather than a theme. Especially in the context of css, calling them `theme` is very confusing.

Reply:
The naming `theme` comes from Figma. If `theme` isn't correct or `theme-page` shouldn't we need to discuss this with design?

Reply:
Also a good point. I agree we should discuss this with design to have consistent naming for this both in different components and between figma and storybook.